test/support/resources/channel/calculations/messages_calculation.ex

```diff
@@ -1,20 +1,24 @@
 defmodule AshGraphql.Test.PageOfChannelMessagesCalculation do
   @moduledoc false
-  use Ash.Calculation
+  use Ash.Resource.Calculation
 
   def load(_, _, context) do
-    limit = context[:limit] || 100
-    offset = context[:offset] || 0
+    limit = context.arguments.limit || 100
+    offset = context.arguments.offset || 0
 
     [
       :channel_message_count,
-      messages: AshGraphql.Test.Message |> Ash.Query.limit(limit) |> Ash.Query.offset(offset)
+      messages:
+        AshGraphql.Test.Message
+        |> Ash.Query.limit(limit)
+        |> Ash.Query.offset(offset)
+        |> Ash.Query.select([:type, :text])
```

Comment:
I had to explicitly add a `select` here, otherwise the load would not load the fields. Not sure if this is normal or if this could be a bug in Ash.

Reply:
Now that I think about it maybe what is missing here is `Ash.Query.new()`

Reply:
Ok it's normal
https://hexdocs.pm/ash/3.0.0-rc.5/upgrade.html#calculation-loads-do-not-select-all-related-fields-by-default